refactor(oxfmt): Arrange and fix main files

leaysgur opened the pull request:
- Rename `run.rs` to `main_napi.rs`
  - `main.rs` is the entrypoint for Rust CLI, and `main_napi.rs` is for JS CLI
- Fix up `cli.ts` to use `.exitCode`, rather than `.exit()`

Boshen commented:
### Merge activity

* **Nov 17, 4:06 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Nov 17, 4:06 AM UTC**: `Boshen` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

leaysgur committed:
refactor(oxfmt): Arrange and fix main files (#15758)

- Rename `run.rs` to `main_napi.rs`
  - `main.rs` is the entrypoint for Rust CLI, and `main_napi.rs` is for JS CLI
- Fix up `cli.ts` to use `.exitCode`, rather than `.exit()`
